QuizMaven

```diff
@@ -0,0 +1,92 @@
+QUIZ
+Note: You could verify your quiz results online https://www.tutorialspoint.com/maven/maven_online_quiz.htm but please
+be honest and answer to this questions before getting to an answer from the site provided.
+
+1.	What of the following is true about POM?
+
+•	It is fundamental Unit of Work in Maven.
+•	It is an XML file.
+•	Both of the above
+•	None of the above.
+
+ANSWER: Both of the above
+
+2.	Which of the following configuration element is present in POM.xml?
+
+•	developers
+•	mailing list
+•	Both of the above.
+•	None of the above.
+
+ANSWER: Both of the above
+
+3.	Which of the following phase in maven life cycle compiles the source code of the project?
+
+•	validate
+•	compile
+•	test
+•	package
+
+ANSWER: compile
```

Comment:
compile, test and package - *ALL* compiles the source code. Later phases like `test` or `package` contains earlier phases like `compile`

Reply:
yes, bur in this question we should choose just one variant. So I've checked compile - compile the source code of the project
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html